server/shared/middleware/validation.js

```diff
@@ -80,6 +74,8 @@ exports.validationError = (error, req, res, next) => {
       return acc;
     }, {});
 
+  if (req.app.get('env') === 'development') console.error(error);
```

Comment:
@eliascodes I'm guessing this will need changing with #402. Might make more sense to merge this first _pending review_, and fix it on your branch, as I was sneakily slipping this in anyway...